build.assets/Dockerfile-centos7

```diff
@@ -126,6 +119,8 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --pr
 # Do this last to take better advantage of the multi-stage build.
 USER root
 COPY --from=libfido2 /usr/local/include/ /usr/local/include/
+COPY --from=libfido2 /usr/local/lib/pkgconfig/ /usr/local/lib/pkgconfig/
+COPY --from=libfido2 /usr/local/lib64/pkgconfig/ /usr/local/lib64/pkgconfig/
```

Comment:
Does pkgconfig always exist in both lib and lib64?

Reply:
Yes, the libs we compiled earlier add it. AFAIK it has no restrictions, depends on where your libs go to.

Reply:
Oddly it doesn't on my openSUSE install but I suppose we don't really need to care about weird installs.

Reply:
I think YMMV a lot in this. We could probably do something like `make LIBDIR=/usr/local/lib64` in libudev to get them all to the same folder. Let me check.

Reply:
Done, looks a bit nicer.

Reply:
What other things are there in `/usr/local/lib` and `/usr/local/lib64` on centos? Is there a way to make it so that we only need a single `.pc` file, or would they be different regardless?

Reply:
> What other things are there in /usr/local/lib and /usr/local/lib64 on centos?

/usr/local/lib is now empty

> Is there a way to make it so that we only need a single .pc file, or would they be different regardless?

I could write the entire Centos7 incantation on /usr/local/lib64/pkgconfig/libfido2-static.pc if we so desire. Currently we have libcrypto-static.pc and libfido2-static.pc in that same folder.

Reply:
No, I meant a single `libfido2-static` for both the regular and the centos7 buildboxes; do we even need `libcrypto-static` as a pkg-config thing? I think we can just roll the `libcrypto.a` path in `libfido2-static.pc` (one for each buildbox, I suppose 😢)

Reply:
Discussed offline.

Splitting libcrypto-static into its own file comes in handy when cross-compiling to other archs (it flips between x86_64-linux-gnu and the other arch folders). We could also "inline" libfido2-static.pc in the Dockerfile, but I think it's mostly cosmetic at this stage, so going ahead as-is.